lnbits/core/crud.py

```diff
@@ -5,8 +5,8 @@
 from uuid import UUID, uuid4
 
 import shortuuid
+from bolt11.decode import decode
```

Comment:
- why not use `lnbits.bolt11` import?

Reply:
- same in `cln.py` and `fake.py`

Reply:
`lnbits.bolt11` is only a compat wrapper for extensions that used it, it should be avoided.

Reply:
> it should be avoided
- why? I think it should be the other way around (use the wrapper if you already have it)
- if `bolt11` lib is replaced with some other lib, or if we want to do some pre/post processing then this wrapper is very useful
- one only needs to change `lnbits.bolt11` and not touch the rest of the codebase

Reply:
@dni ⬆️ ?